call-a-function-api-oci-clouddev.md

```diff
@@ -0,0 +1,772 @@
+---
+layout: collection
+title: Call a Function using API Gateway
+categories: [clouddev, cloudapps, enterprise]
+description: Using Oracle functions to process data via an Oracle API Gateway, and creating a python function to extract HTTP information.
+thumbnail: assets/call-a-func-oracle-funcs-api-gtw-diagram.png
+sort: desc
+tags: [python, apigateway, oci]
+parent: tutorials
+date: 2021-11-20 12:01
+
+---
+{% slides %}
+In this tutorial, you will use Oracle Functions to process data passed from an Oracle API Gateway. You will create a Python function that uses the runtime context to extract HTTP information passed in a request.
+
+Key tasks include how to:
+
+* Gather required information.
+* Create an application for your function.
+* Create a "Hello World!" function.
+* Convert your function to process runtime context data.
+* Deploy and test your function.
+* Create an API Gateway for your function
+* Call your function from the internet using your API Gateway.
+![] {% img [classes] assets/call-a-func-oracle-funcs-api-gtw-diagram.png [1200 675] "" "Oracle Cloud Infrastructure Diagram showing a function executing on the service, the API gateway getting the request and returning the function, with registry in tenancy and VCN within the tenancy allowing access to the Internet" %}(https://docs.oracle.com/en-us/iaas/developer-tutorials/tutorials/functions/func-api-gtw/images/)
```

Comment:
The `{% img %}` tag does not need (and should not have) any other markup around it. The `![]` and trailing parenthetical are incorrect and should be removed.

The `[classes]` should either be replaced with class names such as `alignleft` or `alignright`, or removed entirely.

The dimensions should not be surrounded in square brackets

Are you trying to make the image description show up as a caption or an alt tag? Ideally you'd define both, the first quote being caption, second one being alt tag. If you don't need a caption, only include one quoted string. But you should always have an alt tag. The empty quote pair indicates you were trying to avoid one or the other, but that's not ideal.